src/Core/Services/Implementations/LaunchDarklyFeatureService.cs

```diff
@@ -8,12 +9,38 @@ public class LaunchDarklyFeatureService : IFeatureService, IDisposable
     private readonly LdClient _client;
 
     public LaunchDarklyFeatureService(
-        GlobalSettings globalSettings)
+        IGlobalSettings globalSettings)
     {
-        var ldConfig = Configuration.Builder(globalSettings.LaunchDarkly?.SdkKey)
-                .Offline(globalSettings.SelfHosted)
-                .Build();
-        _client = new LdClient(ldConfig);
+        var ldConfig = Configuration.Builder(globalSettings.LaunchDarkly?.SdkKey);
+
+        if (string.IsNullOrEmpty(globalSettings.LaunchDarkly?.SdkKey))
+        {
+            // support a file to load flag values
+            const string flagOverridePath = "flags.json";
```

Comment:
Why not add this to the launch darkly settings with a default value of `flags.json`?

At the least it seems likely we'll want to be able to update the path easily.